training/06-inner-blocks.md

```diff
@@ -187,15 +168,15 @@ To rig this up, let's go to our [`markup.php`](https://github.com/10up/gutenberg
 
 ```php title="markup.php"
 /**
- * the $args['content'] is the html generated from innerBlocks
+ * the $content is the html generated from innerBlocks
```

Comment:
[It seems that this is being provided as a variable instead](https://github.com/10up/gutenberg-lessons/blob/trunk/themes/tenup-theme/includes/blocks.php#L65) (I get an error if I try to use $args[çontent']).